buildSrc/version.properties

```diff
@@ -6,7 +6,7 @@ bundled_jdk = 13.0.1+9
 
 # optional dependencies
 spatial4j         = 0.7
-jts               = 1.15.0
+jts               = 1.16.1
```

Comment:
I suspect we want to keep this in sync with the version [Lucene is using ](https://github.com/apache/lucene-solr/blob/master/lucene/ivy-versions.properties#L300), which is currently 1.15.0, but then again I don't know if an update there is apropriate as well.